--- /usr/share/X11/xkb/rules/evdev	2020-05-07 11:47:25.891615456 -0400
+++ /home/pat/.xkb/rules/evdev	2020-05-07 11:46:38.908491169 -0400
@@ -1136,6 +1136,7 @@
   lv5:lwin_switch_lock_cancel     =       +level5(lwin_switch_lock_cancel)
   lv5:rwin_switch_lock_cancel     =       +level5(rwin_switch_lock_cancel)
   parens:swap_brackets   =   +parens(swap_brackets)
+  supermods:w   =   +supermods(w)
 
 
 ! option	=	compat
